### ISSUE TITLE
Debugging tests

### ISSUE THREAD

**Issue by virus2016** [maty21/mocha-sidebar#137]
### Description

Cant run in debug mode. I can run the normal 'play' and all tests pass.

#### Steps to Reproduce

1. Hit debug on a test

#### Expected Results

No error 
#### Actual Results

Error: ENOENT: no such file or directory, access 'ts-node/register'
    at Object.accessSync (fs.js:191:3)
    at existsSync (fs.js:222:8)
    at Command.program.on.mod (/Users/calummacrae/.vscode/extensions/maty.vscode-mocha-sidebar-0.20.17/node_modules/mocha/bin/_mocha:330:15)
    at Command.emit (events.js:187:15)
    at Command.parseOptions (/Users/calummacrae/.vscode/extensions/maty.vscode-mocha-sidebar-0.20.17/node_modules/commander/index.js:726:14)
    at Command.parse (/Users/calummacrae/.vscode/extensions/maty.vscode-mocha-sidebar-0.20.17/node_modules/commander/index.js:471:21)
    at Object.<anonymous> (/Users/calummacrae/.vscode/extensions/maty.vscode-mocha-sidebar-0.20.17/node_modules/mocha/bin/_mocha:344:9)
    at Module._compile (internal/modules/cjs/loader.js:686:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
#### Settings file

<!-- Example: {
```
  "mocha.files.glob": "test/**/*.js",
  "mocha.sideBarOptions": {
      "lens": true,
      "decoration": true,
      "autoUpdateTime": 2000,
      "showDebugTestStatus": true
  }
}
``` -->
```
{
    // Mocha: Glob to search for test files
    "mocha.files.glob": "src/**/*.spec.ts",
    //Mocha: run tests on each save
    "mocha.runTestsOnSave": "true",
    // Mocha: Environment variables to run your tests
    "mocha.env": {
        "NODE_ENV": "Test"
    },
    // Mocha: Options to pass to node executable
    "mocha.node_options": [],
    // Mocha: List of files to require before running mocha
    "mocha.requires": [
        "ts-node/register"
    ],
    //Mocha: this option allows you to enable/disable lens decorations and set update threshold "
    "mocha.sideBarOptions": {
        "lens": false,
        "decoration": true,
        "autoUpdateTime": 2000,
        "showDebugTestStatus": true
    }
}
```


### Versions

- VScode: v1.26.1
- Mocha SideBar: v0.20.17
- Node Version: v10.9.0


### OS version
<!-- Check all that apply -->
- [ ] Linux 
- [x] Mac
- [ ] Windows

**Comment by virus2016** [comment]
Okay think I may have found it. 

```
/usr/local/bin/node --inspect-brk=12661 ../../../.vscode/extensions/maty.vscode-mocha-sidebar-0.20.17/node_modules/mocha/bin/_mocha src/**/*.spec.ts --grep '^AuthController Controller should exist$' -r ts-node/register 
```

Looks like the Env don't get passed into the debug mode?

**Comment by maty21** [comment]
hi
you right I missed it 
already fix it I will add it to the next version

**Comment by virus2016** [comment]
Thanks @maty21, Great work btw!

**Comment by maty21** [comment]
Thanks it's already available in the marketplace version 0.20.18
